### DISCUSSION TITLE
Any developing good first issue for beginner?

### DISCUSSION THREAD

**Discussion by noneback** [lf-edge/ekuiper#1836] [General]
I am interested in that IOT streaming project and want to play a role in it. Is there any good first issue to start with? Hopefully, it can help me understand how ekuiper's modules work and interact with each other.

**Comment by ngjaying** [comment]
Warm welcome. We will label good-first-issue but now there is none. We will try to dig some out and label them.
Maybe you can check the test coverage and add some unit tests for low coverage codes?

**Reply by noneback** [reply]
Sure. I will add some ut in some modules that interest me, and try to get familiar with them.

**Reply by ngjaying** [reply]
Cool, looking forward to your contribution
